build/build-debian.xml

```diff
@@ -33,7 +33,13 @@
 			<data src="debian/zap" type="file">
 				<mapper type="perm" strip="1" prefix="/usr/bin" filemode="755"/>
 			</data>
+			<data src="debian/owasp-zap.desktop" type="file">

```

Comment:
This build file does not seem to be in use (and working), the deb package is currently built with build.xml file (in `package-linux` target).